application/src/objects/obstacles/abstract-dynamic-obstacle.js

```diff
@@ -31,9 +31,13 @@ AbstractDynamicObstacle.prototype.getConstants = function()
     };
 };
 
-AbstractDynamicObstacle.prototype.createPhysicsBody = function(state)
+AbstractDynamicObstacle.prototype.createPhysicsBody = function(state, rotation)
 {
     state.game.physics.p2.enable(this);
+
+    if (rotation) {
+        this.body.rotation = rotation * Math.PI / 180;
```

Comment:
Ah, okay. I _thought_ I remembered there was something like that, but I couldn't find it. I'll make that change.